Spielregeln.md

```diff
@@ -0,0 +1,82 @@
+# Spielregeln
+
+## Vorwort
+
+Bierball ist ein Sport, in welchem der Spaß aller Spielenden im Vordergrund steht. 
+Mitspieler und Gegner sind zu jeder Zeit mit Respekt zu behandeln.
+
+## § 1 Spielziel
+
+(1) Das Spiel gilt als beendet, sobald ein Team die Getränke komplett geleert hat.
+
+(2) Ein Getränk gilt als geleert, wenn sich das Trinkbehältnis vom Spielenden mit Öffnung nach unten über den Kopf gehalten wurde, ohne dass eine nennenswerte Menge an Flüssigkeit aus dem Behältnis entweicht.
+Sollte zu viel Flüssigkeit entweichen, erhält das Gegnerteam einen Strafschluck (siehe § 6 (1)).
+
+## § 2 Teams
+
+(1) Gespielt wird mit zwei Teams, die jeweils aus mindestens zwei Personen bestehen.
+
+(2) Die Anzahl der Mitspielenden der beiden Teams darf sich nicht um mehr als eine Person unterscheiden.
+
+## § 3 Spielgeräte
+
+(1) Die folgenden Spielgeräte werden zum Spielen benötigt:
+
+1. Ein Wurfgerät
+2. Ein Ziel
+3. Ein Getränkebehältnis pro spielender Person
+
+(2) Das Wurfgerät ist ein annähernd runder Gegensand, der sich leicht einhändig werfen lässt.
+
+(3) Das Ziel muss in Größe und Form einer 1,0-Liter-Plastikflasche ähneln.
+Es muss in einer Art beschwert sein (z.B. durch das Befüllen mit Wasser), dass es nicht durch leichte Krafteinwirkung, wie z.B. Windstöße umfällt.
+Allerdings ist sicherzustellen, dass es möglich ist, das Ziel mit dem Wurfgerät zu Fall zu bringen.
+
+(4) Getrunken wird aus Glasflaschen der 0,5-Liter-Klasse, deren Öffnung DIN EN ISO 12822 oder DIN EN ISO 12821 (handelsübliche Bierflasche mit Kronkorkenverschluss) entspricht.
+Die Getränke müssen Kohlensäure enthalten.
+Es ist explizit erlaubt, Getränke zu konsumieren die keinen Alkohol enthalten bzw. nicht als Bier klassifiziert werden.
+
+## § 4 Spielfeld
+
+(1) In der Mitte des Spielfeldes wird das Ziel aufgestellt.
+
+(2) Von dort ausgehend sind in entgegengesetzte Richtung 6m (~7 Schritte) abzumessen an denen sich die jeweiligen Grundlinien der zwei Teams befinden.
+
+(3) Die Grundlinien der Teams verlaufen parallel.
+
+## § 5 Ablauf
+
+(1) Vor Beginn entscheidet ein Münzwurf oder eine Partie Schere-Stein-Papier zwischen zwei Mitgliedern der spielenden Teams über den ersten Wurf.
+Das gewinnende Team entscheidet, welches Team zuerst wirft.
+
+(2) Vor jedem Wurf, muss sichergestellt werden, dass das Gegnerteam bereit ist.
+Bei Missachtung dieser Regel wird der Wurf annulliert und das Wurfrecht geht an das Gegnerteam über.
+
+(3) Geworfen wird stehend, wobei der Ball beim Wurf die Wurfhand über Bauchhöhe verlassen muss.
```

Comment:
Wurde das sonst auch so gespielt? Ich dachte Schulterhöhe.

Reply:
Es wurde immer Schulterhöhe gesagt. Allerdings soll die Regel ja vor allem dazu dienen, dass nicht von unten geworfen wird. Ich hatte überlegt die Regel so abzuändern, dass es weniger Diskussion bei leicht unsauberen Würfen gibt. Allerdings könnte Bauchhöhe vermutlich etwas zu niedrig sein. 
Da muss vermutlich noch mal in größerer Runde dsikutiert werden.